### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by rkkripa**
Made some changes! Check out.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,9 +1,11 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>Hello World</title>
+    <title>Hello </title>
```
Comment (rambo436) [inline index.html]: @rkkripa Was this extra space left here on purpose? ;)

Comment (rkkripa) [inline index.html]: @kaiprt  :D no i guess by mistake, as i just deleted the word "World :P

Comment (rambo436) [inline index.html]: Lol, remove the whitespace, make a commit, and push to this branch again. You will see the pull request update itself and I will review again. :dancer:

Comment (rkkripa) [inline index.html]: Haha Cool :dancers:  will do this :+1:

**Comment by rambo436** [comment]
:+1:  @rkkripa You can now merge into the master branch.

**Comment by rkkripa** [comment]
Cool :+1:  :dancer:

**Pull request merged**